### ISSUE TITLE
Handle processing on concatenated scans per session

### ISSUE THREAD

**Issue by ttngu207** [datajoint/element-calcium-imaging#15]
Currently, all downstream processing (motion correction + segmentation) are performed at a per scan level (`imaging.Scan`). 
For many users, a typical routine is to concatenate data from multiple scans in a particular session and then run processing (e.g. run through Suite2p). 
We need to modify/upgrade the pipeline to handle this. One possible solution is to have the `imaging.ProcessingTask` reference the `Session` table instead of `scan.Scan` table, and have another part table to specify which scans go into this ProcessingTask

```
@schema
class ProcessingTask(dj.Manual):
    definition = """
    -> Session
    -> ProcessingParamSet
    scan_ids: varchar(32)  # list of the scan_id associated with this ProcessingTask (delimited by '_' for example)
    ---
    task_mode='load': enum('load', 'trigger')  # 'load': load computed analysis results, 'trigger': trigger computation
    """

        class MemberScan(dj.Part):
            definition = """
                -> master
                -> scan.Scan
            """

**Comment by ttngu207** [comment]
However, the proposed approach above cannot fully solve the problem yet. There are many tables downstream from the processing that reference back to the scan being processed - e.g. Summary images, segmented Masks, etc. point back to the `Scan.Field` they are processed from.

Technically speaking, all of the concatenated `Scan` should have the same structure (e.g. same number of fields, depths, size, location, etc.), since the concatenation should be across time only (and we can check for this). So, downstream tables technically can point to any of the Scan (in the `MemberScan`) and the meta info is still correctly preserved, but still we need to specify one Scan to point to anyhow. 

Should we randomly pick one? or pick the first one in the list? How should this be communicated so that users understand that referencing any of these scans are all equivalent to each other?

**Comment by kabilar** [comment]
Addressed in [PR#35](https://github.com/datajoint/element-calcium-imaging/pull/35).
